include/mbedtls/ssl_internal.h

```diff
@@ -238,7 +238,7 @@
    implicit sequence number. */
 #define MBEDTLS_SSL_HEADER_LEN 13
 
-#if defined(MBEDTLS_SSL_DTLS_CONNECTION_ID)
+#if !defined(MBEDTLS_SSL_DTLS_CONNECTION_ID)
```

Comment:
In commit "Fix I/O buffer sizes when MBEDTLS_SSL_DTLS_CONNECTION_ID is enabled", describe in the git commit message, on its own line, that the commit "Fixes #3077"